src/components/dialogs/ResolutionDateDialog.vue

```diff
@@ -9,12 +9,14 @@
           have passed a resolution or have a court order to change your share structure.</p>
           <p>Enter the date of the Resolution or Court Order here:</p>
         </div>
-        <date-picker title="Resolution or Court Order Date"
-                     :error-msg="errors"
-                     nudge-right="100"
-                     @emitDate="onDateEmitted($event)"
-                     @emitCancel="exit()"
-                     @emitDateSync="date = $event"/>
+        <date-picker
+          title="Resolution or Court Order Date"
+          :error-msg="errors"
+          nudge-right="100"
+          @emitDate="onDateEmitted($event)"
+          @emitCancel="exit()"
+          @emitDateSync="date = $event"
+        />
```

Comment:
Could you please confirm with Linda and if OK then add a maxDate (today) here? (This requirement came after my ticket was implemented.)

Thanks for updating this page BTW 👍

Reply:
I believe there is was much more work to come here with regards to these dates, following up tho!

Reply:
Ah, ok. If it's not small then I suggest a new ticket for that.

Reply:
ah yes, work is captured here: https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/6904

Certainly covers a few things, so we can wait on applying them here.

Reply:
OK. Good to merge when ready then.